docker_services_cli/config.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C 2020 CERN.
+#
+# Docker-Services-CLI is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Config module."""
+
+DOCKER_SERVICES_FILEPATH = "docker_services_cli/docker-services.yml"
+
+DEFAULT_VERSIONS = {
+    "ES_5_LATEST": "5.6.10",
+    "ES_6_LATEST": "6.8.12",
+    "ES_7_LATEST": "7.9.0",
+    "POSTGRESQL_9_LATEST": "9.6.19",
+    "POSTGRESQL_10_LATEST": "10.14",
+    "POSTGRESQL_11_LATEST": "11.9",
+    "MYSQL_5_LATEST": "5.7.31",
+    "MYSQL_8_LATEST": "8.0.21",
+    "REDIS_6_LATEST": "6.0.6",
+    "MEMCACHED_LATEST": "1.6.6",
+    "RABBITMQ_3_LATEST": "3.8.7",
+}
+
+# Elasticsearch
+ELASTICSEARCH = {
+    "ES_VERSION": "ES_7_LATEST"
+}
```

Comment:
I think it would be worth adding docsstrings to this. My first reaction after reading this and having no context would be: isn't this wrong? shouldn't it be the following instead?
```python
ELASTICSEARCH = {
    "ES_VERSION": DEFAULT_VERSIONS["ES_7_LATEST"]
}
```
And in fact it is correct because it is managed through environment variables.